components/admin/company-selector.tsx

```
@@ -0,0 +1,232 @@
+				</CardBody>
+			</Card>
+
+			{/* Backdrop to close dropdown */}
+			{isDropdownOpen && (
+				<div
+					className="fixed inset-0 z-40"
+					onClick={() => {
+						setIsDropdownOpen(false);
+						setSearchTerm('');
+					}}
+				/>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)